src/my_functions_product1.py

```diff
@@ -0,0 +1,721 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jun 11 15:15:42 2018
+
+@author: bursaliogluozgun
+"""
+import pandas as pd
+import numpy as np
+
+
+import keras as ke
+import keras.backend as K
+from keras.layers import Input, Dense, Dropout
+from keras.models import Model
+from keras.models import load_model
+# model reconstruction from JSON:
+from keras.models import model_from_json
+
+
+import sklearn as sk
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.metrics import accuracy_score, roc_auc_score
+from sklearn.utils.class_weight import compute_class_weight
+#==============================================================================
+def Input_file_checking(filename):
+    """ 
+       This function checks if the input file exists
```

Comment:
Please add doc-strings. Also go over variable naming conventions in that doc (I was aware of the pep8 doc! i'm not specific about naming conventions, but others might be. There are different conventions across languages!)
Doc-string general format: 
"""A one-line description of the function 
:param filename: .....
:return: ...."""